### ISSUE TITLE
proto3 version

### ISSUE THREAD

**Issue by ZhangHongjuan** [wangguojun2018/LidarRoadBoundaryDetection#3]
Hi, thanks for your work, it's really awesome. But when I installed it, it failed with an error information "/road_boundary.pb.h:17:2: error: #error This file was generated by an older version of protoc which is". I used protobuf-3.15.0 and I would like to ask which specific version of protoc did you use?

**Comment by goksanisil23** [comment]
hi, you can generate the proto files yourself if you just delete the .cc and .h files inside the proto folder.

**Comment by crankler** [comment]
Hi, can you tell me how to  generate the proto files?

**Comment by crankler** [comment]
Yes, I have solve it,  use the example command , and I can get the new file
> protoc --cpp_out=./ project.proto
